src/status_im/contact/core.cljs

```diff
@@ -151,3 +151,8 @@
    :mutual-contact-enabled?
    enabled?
    nil))
+
+(fx/defn set-search-query
+  {:events [:contacts/set-search-query]}
+  [{:keys [db] :as cofx} value]
+  {:db (assoc db :contacts/set-search-query value)})
```

Comment:
only event :( key in the db should be without set, so event `:contacts/set-search-query` sets the value for `:contacts/search-query`